mappings/net/minecraft/world/loot/LootTableContext.mapping

```diff
@@ -1,14 +1,58 @@
 CLASS none/bbk net/minecraft/world/loot/LootTableContext
 	CLASS none/bbk$a Builder
 		FIELD a world Lnone/lu;
+		FIELD b luck F
+		FIELD c lootedEntity Lnone/sg;
+		FIELD d player Lnone/aam;
+		FIELD e source Lnone/rr;
 		METHOD <init> (Lnone/lu;)V
 			ARG 0 world
 		METHOD a build ()Lnone/bbk;
+		METHOD a setLuck (F)Lnone/bbk$a;
+			ARG 0 luck
+		METHOD a setPlayer (Lnone/aam;)Lnone/bbk$a;
+			ARG 0 player
+		METHOD a setDamageSource (Lnone/rr;)Lnone/bbk$a;
+			ARG 0 source
+		METHOD a setLootedEntity (Lnone/sg;)Lnone/bbk$a;
+			ARG 0 lootedEntity
+	CLASS none/bbk$b TARGET
+		CLASS none/bbk$b$a Serializer
```

Comment:
The class is named `TARGET` or is it an anonymous class?
Also `DeSerializer` to go with our current naming scheme.